src/test/java/com/xjeffrose/xio/fixtures/OkHttpUnsafe.java

```diff
@@ -16,6 +16,8 @@
 import javax.net.ssl.X509TrustManager;
 import okhttp3.OkHttpClient;
 import okhttp3.mockwebserver.MockWebServer;
+import java.util.Arrays;
```

Comment:
Sort?

Reply:
I'm surprised the pre-commit git-hook missed that.